scripts/exregional_run_fcst.sh

```diff
@@ -512,6 +515,21 @@ POST_STEP
 #
 #-----------------------------------------------------------------------
 #
+# Move RESTART directory to COMIN and create symlink in DATA only for
+# NCO mode and when it is not empty.
+#
+#-----------------------------------------------------------------------
+#
+if [ "${RUN_ENVIR}" = "nco" ]; then
+  rm_vrfy -rf "${COMIN}/RESTART"
+  if [ "$(ls -A RESTART)" ]; then
+    mv_vrfy RESTART ${COMIN}
+    ln_vrfy -sf ${COMIN}/RESTART ${DATA}/RESTART
+  fi
+fi
+#
```

Comment:
Same questions as above.

Reply:
I see from the PR description that this is for cycled runs and you did this most likely because RESTART is not accessible because it is in forecast directory which has PID attached to it. That is no more the case in PR #443 , which makes the forecast directory accessible to all other tasks by not using PID but another unique ID known to all other tasks. So with this new implementation there is no need to symlink RESTART to COMIN since the forecast directory is accessible still after restart.